### DISCUSSION TITLE
install the finn framework using docker desktop through windows cmd

### DISCUSSION THREAD

**Discussion by JaouaherBelgacem** [Xilinx/finn#934] [Q&A]
I am new to the finn framework, and I did not understand how to install the finn framework with docker using the cmd command as the commands are not clear.

**Comment by fpjentzsch** [comment]
Hi, FINN does not officially support Windows. I recommend Ubuntu 20 or 22. Then you can simply follow the Getting Started guide: https://finn.readthedocs.io/en/latest/getting_started.html

You could also run FINN using Ubuntu and Docker Desktop on Windows via WSL 2.

**Reply by JaouaherBelgacem** [reply]
Hi, thank you for your reply! I installed Ubuntu and docker desktop using the wsl2 and they are working through the PowerShell. However, when I tried to clone the GitHub repository it was not possible using the running Ubuntu on wsl2, however when I use the power shell terminal, the finn repository exists. I can list its content but not with Ubuntu and this is what I get:
git clone https://github.com/Xilinx/finn/
Cloning into 'finn'...
fatal: unable to access 'https://github.com/Xilinx/finn/': Could not resolve host: github.com
I enabled the VPN on wsl and I tried to check the problem online but the suggested solution didn't solve the problem

**Reply by fpjentzsch** [reply]
Mhh, sounds like a DNS issue. Although I have never had trouble with the outgoing internet connection from within the WSL 2 distribution.

You should not need to use PowerShell except for the initial install of WSL 2. To access the Linux shell I recommend to use the new "Windows Terminal", which you can install via the Microsoft Store (if it is not installed by default by now).

**Reply by JaouaherBelgacem** [reply]
I installed the Windows terminal preview app and I used wsl2 through it
